DTML.EduBot/Controllers/MessagesController.cs

```diff
@@ -18,6 +21,14 @@ public class MessagesController : ApiController
         {
             if (activity.Type == ActivityTypes.Message)
             {
+                var userQuery = activity.Text;
```

Comment:
I think this whole translate API call should not happen within the controller layer. It deserves its own service layer, like QnA Service.